packages/alfa-style/src/property/display.ts

```diff
@@ -76,54 +83,172 @@ export namespace Specified {
  */
 export type Computed = Specified;
 
+/**
+ * {@link https://drafts.csswg.org/css-display/#typedef-display-outside}
+ */
+const parseDisplayOutside = Keyword.parse("block", "inline", "run-in");
+
+/**
+ * {@link https://drafts.csswg.org/css-display/#typedef-display-inside}
+ */
+const parseDisplayInside = Keyword.parse(
+  "flow",
+  "flow-root",
+  "table",
+  "flex",
+  "grid",
+  "ruby"
+);
+
+/**
+ * {@link https://drafts.csswg.org/css-display/#typedef-display-listitem}
+ */
+const parseDisplayListItem = Keyword.parse("list-item");
```

Comment:
Do we need to repeat `Display` on the names of these functions? They're already namespaced and anyway not exported.

Reply:
Nope, I'll remove them 👍